packages/shared/src/components/sidebar/sections/CustomFeedSection.tsx

```diff
@@ -38,6 +40,12 @@ export const CustomFeedSection = ({
             icon: feed.node.flags.icon || (
               <HashtagIcon secondary={isCustomFeedPageActive} />
             ),
+            rightIcon: () => (
+              <StarIcon
+                secondary
+                className="text-surface-disabled opacity-[0.8]"
```

Comment:
Isn't one other text-surface already including the opacity?
Normally shouldn't have to use it as such.

The surface disabled already has transparency as well.

Reply:
It looks different.. and in figma it is specifically set to that.. I'll check with Tsahi

Reply:
adjusted to what it is in our system